### ISSUE TITLE
Problem: no way to calculate stake weight from the CLI at for a ledger

### ISSUE THREAD

**Issue by trevorbernard** [Granola-Team/mina-indexer#524]
e.g. `mina-indexer staking-weight --public-key <b62...>`

**Comment by trevorbernard** [comment]
This isn't as easy as it first appeared. Is there value in knowing the current stake weight for a block producer? IMHO, I suspect very little. @robinbb thoughts?

**Comment by robinbb** [comment]
Yes, I would say this is very interesting and useful information. It should be easy to get this info, if we are ingesting staking ledgers logged from the Mina node.
